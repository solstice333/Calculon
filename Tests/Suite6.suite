P a.out compile.c
T test10.in test10.out 1000
P b.out compile.c
T test10.in test10.out 1000
