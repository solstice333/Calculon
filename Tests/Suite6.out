compile.c:1: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘is’
compile.c:2: error: expected identifier or ‘(’ before ‘<’ token
compile.c:2: error: stray ‘#’ in program
compile.c:3: error: expected identifier or ‘(’ before ‘++’ token
compile.c:3:3: error: empty character constant
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
Failed: gcc compile.c -o a.out
compile.c:1: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘is’
compile.c:2: error: expected identifier or ‘(’ before ‘<’ token
compile.c:2: error: stray ‘#’ in program
compile.c:3: error: expected identifier or ‘(’ before ‘++’ token
compile.c:3:3: error: empty character constant
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
compile.c:4: error: expected identifier or ‘(’ before ‘{’ token
Failed: gcc compile.c -o b.out
